### DISCUSSION TITLE
Error on initial run of .\webui.bat

### DISCUSSION THREAD

**Discussion by Rainz3** [vladmandic/sdnext#472] [Q&A]
I am receiving the following two errors on a fresh install.

PS H:\Stable nividia\automatic> .\webui.bat
venv "H:\Stable nividia\automatic\venv\Scripts\Python.exe"
18:26:15-563828 INFO     Python 3.10.9 on Windows
18:26:15-628826 INFO     Version: 82d7b45c Mon Apr 24 16:54:12 2023 -0400
18:26:15-925997 INFO     nVidia toolkit detected
18:26:15-926955 INFO     Installing package: torch torchaudio torchvision --index-url
                         https://download.pytorch.org/whl/cu118
18:26:46-023374 ERROR    Error running pip: install --upgrade torch torchaudio torchvision --index-url
                         https://download.pytorch.org/whl/cu118
18:26:46-135607 ERROR    Could not load torch: [WinError 126] The specified module could not be found. Error loading
                         "H:\Stable nividia\automatic\venv\lib\site-packages\torch\lib\caffe2_nvrtc.dll" or one of its
                         dependencies.

**Comment by vladmandic** [comment]
That's deep inside torch, not sure how much we can do about that.
Just to rule out possible silly torch issue, can you try installing into folder without spaces in name?

**Reply by Rainz3** [reply]
Tried renaming the folder and got the same results. Tried reinstalling pytorch and the cuda drivers and still getting the same error. I do some more digging.
